tests/Message/MessageBagTest.php

```diff
@@ -56,7 +59,11 @@ public function with(): void
 
         self::assertCount(3, $messageBag);
         self::assertCount(4, $newMessageBag);
-        self::assertSame('It is time to wake up.', $newMessageBag[3]->content);
+
+        $newMessageFromBag = $newMessageBag[3];
+        assert($newMessageFromBag instanceof AssistantMessage);
```

Comment:
can we use the assertion method of PHPUnit for such things please?

Reply:
like
```suggestion
        self::assertInstanceOf(AssistantMessage::class, $messageFromBag);
```

Reply:
Sure! Done.